codegen/smithy-aws-typescript-codegen/src/main/java/software/amazon/smithy/aws/typescript/codegen/JsonShapeSerVisitor.java

```diff
@@ -55,16 +55,19 @@ final class JsonShapeSerVisitor extends DocumentShapeSerVisitor {
 
     private final BiFunction<MemberShape, String, String> memberNameStrategy;
 
-    JsonShapeSerVisitor(GenerationContext context) {
+    JsonShapeSerVisitor(GenerationContext context, boolean serdeElisionEnabled) {
```

Comment:
interface change is ok, right? if any external consumers upgrade, they change their calls at the same time?